src/arr/compiler/js-dag-utils.arr

```diff
@@ -339,25 +339,16 @@ fun stmts-of(blk :: J.JBlock):
   end
 end
 
-fun find-steps-to(stmts :: ConcatList<J.JStmt>, step :: A.Name, acc :: D.MutableStringDict<J.Label>) -> D.MutableStringDict<J.Label>:
+fun find-steps-to(stmts :: ConcatList<J.JStmt>, step :: A.Name, acc :: D.MutableStringDict<J.Label>, cases-dispatches :: ConcatList<J.JStmt>) -> D.MutableStringDict<J.Label>:
   var looking-for = none
```

Comment:
I think you can nuke this variable now, too, right?  If so, hooray!  It was always an ugly hack...